src/pages/search/component.js

```diff
@@ -0,0 +1,81 @@
+import React, { useState, useEffect } from 'react'
+import style from './style.module.scss'
+// ROUTER
+import { useNavigate, useLocation } from 'react-router-dom'
+
+const Search = () => {
+    // STATE
+    const [movie, setMovie] = useState(null)
+	const [error, setError] = useState(null)
+	const [isPending, setIsPending] = useState(false)
+    const navigate = useNavigate()
+
+    // extract the search query parameters
+    const queryString = useLocation().search;
+    const queryParams = new URLSearchParams(queryString)
+    const query = queryParams.get('query')
+
+    // useEFFECT
+    const fetchMovie = async () => {
+        setIsPending(true)
+
+		try{
+			const response = await fetch(`http://localhost:4000/movies?search=${query}&searchBy=title`)
```

Comment:
since you already have React, all this stuff would be done thought it
getting the data will be done with thunk, stored in store and just being drugged from the store with the selector here